gprofiler/metadata/application_identifiers.py

```diff
@@ -204,6 +204,14 @@ def get_application_name(self, process: Process) -> Optional[str]:
         app_name = _get_cli_arg_by_name(process.cmdline(), "-A") or _get_cli_arg_by_name(
             process.cmdline(), "--app", check_for_equals_arg=True
         )
+        if app_name is _NON_AVAILABLE_ARG:
+            queue_name = _get_cli_arg_by_name(process.cmdline(), "-Q") or _get_cli_arg_by_name(
+                process.cmdline(), "--queues", check_for_equals_arg=True
+            )
+            # TODO: One worker can handle multiple queues, it could be useful to encode that into the app id.
+            if queue_name is not _NON_AVAILABLE_ARG:
+                # The queue handler routing is defined in the directory where the worker is run
+                return f'celery queue: {queue_name} ({process.cwd()})'
```

Comment:
Does it make sense to add cwd?

We add it for script/file-based appids because the cwd makes a difference (i.e `python app.py` depends on cwd). Does it matter with queues?

Reply:
Yes, I wrote a comment in the line above

Reply:
Lol